### PR TITLE
Out with scss_lint going with stylelint

### PR DESCRIPTION
**Pull request by xipasduarte**
Putting stuff up to date with the latest changes in @goblindegook's [genesis-starter](https://github.com/goblindegook/genesis-starter) repository.

### CODE REVIEW

**Comment by xipasduarte** [comment]
The dot file `.stylelintrc` is already in the repository's root.

**Comment by xipasduarte** [comment]
Still missing: correcting what the linter found.

I'm saving this for this week's down time intervals (meaning I'll do it next Saturday/Sunday 😄 ). Hopping to push something else into the mix here. Stay tuned...

File: package.json [hunk 1]
```diff
@@ -21,71 +21,79 @@
       "underscore": "lodash"
     }
   },
+  "ava": {
```
Comment (goblindegook) [inline package.json:24]: I'd probably ditch AVA for a combination of Tape/Jasmine2/Chai + Karma. (Add Enzyme if testing React components.) See goblindegook/littlefoot for an example using Tape + Karma.

Comment (xipasduarte) [inline package.json:24]: Wait for it then. Would you like me to feed this back to goblindegook/genesis-starter?

**Pull request merged**